### PR TITLE
Added issue template

### PR DESCRIPTION
**Pull request by sahilarora3117**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+**Description**
+
+**To Reproduce**
+Also attach debug dump.
+
+```
+./Binary_Name --gtk-debug=all
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The command is `G_MESSAGES_DEBUG=all com.github.arshubham.gitignore`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description**
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Add check boxes to ask if issue is a bug/feature request/other

**Comment by sahilarora3117** [comment]
There are other type of templates for feature request and other issues. So will have to add it .GitHub folder

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,22 +1,25 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a report to help us improve.
 title: ''
 labels: ''
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Add 'bug' here Will automatically add the label

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
+Explain your problem.  
 
-**To Reproduce**
-Also attach debug dump.
+**To Reproduce**  
+Execute the program using the following command and attach the data.
 
 ```
-./Binary_Name --gtk-debug=all
+ G_MESSAGES_DEBUG=all com.github.arshubham.gitignore
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `Use single back-ticks here.`

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/feature_request.md]: add 'enhancement' here

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (sahilarora3117) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Unable to understand this. Please explain.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: Suggest enhancement for this project.
+about: Enhancement
```
Comment (arshubham) [inline .github/ISSUE_TEMPLATE/feature_request.md]: use small e here

**Pull request merged**